pkg/config/remote/service/service.go

```diff
@@ -138,8 +139,8 @@ func NewService() (*Service, error) {
 	}
 
 	clientsTTL := time.Second * config.Datadog.GetDuration("remote_configuration.clients.ttl_seconds")
```

Comment:
This should be either just GetDuration, or `time.Second * GetInt`. Multiplying the result of GetDuration produces incorrect results when the configuration is given in seconds (`30s`), like the default you are changing. (It is unfortunate that go allows dimensionless durations and defaults to a unit that makes this code work in some cases).

Reply:
couldn't agree more, C++ with std::chrono::duration made it right